pkg/analyze/ceph.go

```diff
@@ -173,3 +227,18 @@ func compareCephStatus(actual, when string) (bool, error) {
 		return false, errors.New("unknown operator")
 	}
 }
+
+func detailedMessage(msg string, status CephStatus) string {
+	if status.OsdMap.OsdMap.Full {
+		msg = fmt.Sprintf("%s: OSD is full", msg)
```

Comment:
i might change the ":" because i don't see any other cases where use that delimiter. "." might make more sense.

Reply:
ended up making this change